dot/rpc/modules/chain_test.go

```diff
@@ -23,17 +23,71 @@ func newChainService(t *testing.T) *state.Service {
 
 	tr := trie.NewEmptyTrie()
 
+	stateSrvc.UseMemDB()
+
 	err = stateSrvc.Initialize(genesisHeader, tr)
 	if err != nil {
 		t.Fatal(err)
 	}
+
 	err = stateSrvc.Start()
 	if err != nil {
 		t.Fatal(err)
 	}
+
+	err = loadTestBlocks(genesisHeader.Hash(), stateSrvc.Block)
+	if err != nil {
+		t.Fatal(err)
+	}
 	return stateSrvc
 }
 
+func loadTestBlocks(gh common.Hash, bs *state.BlockState) error {
```

Comment:
there's another test function in state called `addBlocksToState` where you can specify the depth (the branching part of that function probably isn't too relevant here) that might be useful in the future for adding more than 2 blocks

Reply:
excellent, I thought there may be something, but I didn't look deeply enough.  Will come in handy.